spec/models/manageiq/providers/openstack/cloud_manager/refresh_spec_common.rb

```diff
@@ -58,7 +58,6 @@ def assert_with_errors
       expect(OrchestrationStackResource.count).to  eq 0
       expect(OrchestrationStackOutput.count).to    eq 0
       expect(OrchestrationTemplate.count).to       eq 0
-      expect(CloudObjectStoreContainer.count).to   eq storage_data.directories.count
```

Comment:
@hsong-rh So, we don't know why this was failing? Removing the tests isn't a good idea. Even it it's temporary we need to ensure they get added back in at some point. Commenting out the test, with a descriptive comment, would make the intent obvious. As it stands, @jerryk55 will have to replace these tests in his PR. Jerry, is that OK with you?